supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -0,0 +1,347 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Product data information for supporters."
+
+Parameters:
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - PROD
+      - UAT
+      - DEV
+    Default: DEV
+
+Conditions:
+  CreateProdResources: !Equals [!Ref "Stage", "PROD"]
+
+Mappings:
+  StageVariables:
+    DEV:
+      S3Bucket: supporter-product-data-export-dev
+      # Every 10 mins between 7am and 7pm Mon-Fri
+      scheduleRate: "cron(*/10 7-19 ? * 1-5 *)"
+    UAT:
+      S3Bucket: supporter-product-data-export-uat
+      scheduleRate: "cron(*/10 7-19 ? * 1-5 *)"
+    PROD:
+      S3Bucket: supporter-product-data-export-prod
+      # Every 5 mins
+      scheduleRate: "cron(*/5 * ? * * *)"
```

Comment:
we schedule every 5 minutes, but the state machine might retry for 20 minutes when fetching the results.  I'm just wondering what would happen if two are doing the incremental query at the same time?